core-feature-pack/common/src/main/resources/content/bin/installation-manager.bat

```diff
@@ -51,24 +58,30 @@ set "INST_MGR_COMMAND=!INST_MGR_COMMAND:\:=:!"
 set "INST_MGR_COMMAND=!INST_MGR_COMMAND:\\=\!"
 setlocal DisableDelayedExpansion
 
-set JAVA_OPTS=-Dlogging.configuration=file:"%INST_MGR_LOG_PROPERTIES%" %JAVA_OPTS%
+set JAVA_OPTS=-Dlogging.configuration=file:"%INST_MGR_LOG_PROPERTIES%" -Dorg.jboss.boot.log.file="%INST_MGR_LOG_FILE%" -Dorg.wildfly.prospero.log.file %JAVA_OPTS%
```

Comment:
@yersan shouldn't the `org.wildfly.prospero.log.file` be set to `INST_MGR_LOG_FILE` ?

Reply:
@spyrkob It could, but the value of `org.wildfly.prospero.log.file` is only used by Prospero logging configuration file and here we are using the server logging configuration file available at `JBOSS_CONFIG_DIR\logging.properties`

This `org.wildfly.prospero.log.file` empty initialization here is just a trick to avoid that `prospero.sh[bat,ps1]` do any configuration that could affect the logging framework. When this property is initialized at the JAVA_OPTS, `prospero.sh[bat,ps1]` will ignore it and won't add any additional JAVA_OPTS configuration, see: https://github.com/wildfly-extras/prospero/blob/main/dist/common/src/main/resources/packages/org.jboss.prospero/content/bin/prospero.sh#L94 

As you said, it could be initialized to "%INST_MGR_LOG_FILE%", but really, it is an unknown property from the point of view of `JBOSS_CONFIG_DIR\logging.properties`. Maybe it is better to give it a value, but it is not used at all.